exchange/utils.go

```diff
@@ -212,6 +212,11 @@ func (rs *requestSplitter) cleanOpenRTBRequests(ctx context.Context,
 			applyFPD(auctionReq.FirstPartyData[bidderRequest.BidderName], bidderRequest.BidRequest)
 		}
 
+		passTIDActivityAllowed := auctionReq.Activities.Allow(privacy.ActivityTransmitTids, scopedName)
+		if !passTIDActivityAllowed {
+			privacyEnforcement.TID = true
+		}
```

Comment:
The Transmit TID activity is new and doesn't have the baggage of TCF, CCPA, etc.. Could this be simplified as:

`privacyEnforcement.TID = !auctionReq.Activities.Allow(privacy.ActivityTransmitTids, scopedName)`